### DISCUSSION TITLE
Is it possible to set log level for .nextflow.log to INFO

### DISCUSSION THREAD

**Discussion by andrimar1** [nextflow-io/nextflow#3635] [Q&A]
Is there any way of configuring the log level to INFO instead of DEBUG. As far as I understand adding the `-quite` flag sets the log level to WARN.

[LoggerHelper.groovy](https://github.com/nextflow-io/nextflow/blob/master/modules/nextflow/src/main/groovy/nextflow/util/LoggerHelper.groovy)

**Comment by pditommaso** [comment]
Think there's no setting for that
